android/src/main/java/com/tailscale/ipn/ui/localapi/LocalAPIClient.kt

```diff
@@ -6,20 +6,35 @@ package com.tailscale.ipn.ui.localapi
 
 import android.util.Log
 import com.tailscale.ipn.ui.model.*
-import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
 
-// A response from the echo endpoint.
 typealias StatusResponseHandler = (Result<IpnState.Status>) -> Unit
-
 typealias BugReportIdHandler = (Result<BugReportID>) -> Unit
-
 typealias PrefsHandler = (Result<Ipn.Prefs>) -> Unit
 
 class LocalApiClient {
     constructor() {
         Log.d("LocalApiClient", "LocalApiClient created")
     }
 
+    companion object {
+        private val _isReady = MutableStateFlow(false)
```

Comment:
This is only going to get set once right? 
Maybe use something like a CompletableDeferred instead?